ONBOARDING.md

```diff
@@ -1,91 +1,90 @@
-- [Introduction](#introduction)
-  - [Who this is for](#who-this-is-for)
-  - [What this guide will help you do](#what-this-guide-will-help-you-do)
-- [Learn key concepts](#learn-key-concepts)
-  - [About the OpenSearch project](#about-the-opensearch-project)
-  - [How we work](#how-we-work)
-  - [Contributor expectations](#contributor-expectations)
-  - [Resources and links](#resources-and-links)
-- [Set up accounts and access](#set-up-accounts-and-access)
-  - [GitHub account](#github-account)
-  - [OpenSearch forum and Slack accounts](#opensearch-forum-and-slack-accounts)
-
-# Getting started as an OpenSearch contributor
+- [Getting Started as an OpenSearch Project Contributor](#getting-started-as-an-opensearch-project-contributor)
+  - [Introduction](#introduction)
+    - [Who is this for?](#who-is-this-for)
+    - [What will this guide help you do?](#what-will-this-guide-help-you-do)
+  - [Key Concepts](#key-concepts)
+    - [About the OpenSearch Project](#about-the-opensearch-project)
+    - [How We Work](#how-we-work)
+    - [Contributor Expectations](#contributor-expectations)
+    - [Resources and Links](#resources-and-links)
+  - [Set Up Accounts and Access](#set-up-accounts-and-access)
+    - [GitHub Account](#github-account)
+    - [OpenSearch Forum and Slack Accounts](#opensearch-forum-and-slack-accounts)
+
+# Getting Started as an OpenSearch Project Contributor
 
 ## Introduction
 
-### Who this is for
+### Who is this for?
 
-Any developer who's interested in contributing code (features and fixes) to one of the OpenSearch open source project repositories for the first time.
+Anyone who's interested in contributing code (features, fixes, documentation) to one of the OpenSearch Project open-source repositories for the first time.
 
-### What this guide will help you do
+### What will this guide help you do?
 
-1. Understand the key concepts of the OpenSearch project and how to find additional resources
-2. Set up the accounts you'll need to contribute
+1. Understanding the key concepts of the OpenSearch Project and how to find additional resources.
```

Comment:
I think if we're answering the question "what will this guide help you do", then "understanding" is better.  But if you want  change it to understanding, then you should change "set up" to "setting up"?

Reply:
Put "Understand" back.